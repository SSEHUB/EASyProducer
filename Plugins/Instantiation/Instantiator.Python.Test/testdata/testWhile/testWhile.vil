vilScript testWhile (Project source, Configuration config, Project target) {

    version v0;

    main(Project source, Configuration config, Project target) = : {   	
    	vilTemplateProcessor("$target/testWhile.vtl", config, "$target/testWhile.py");
    }
}