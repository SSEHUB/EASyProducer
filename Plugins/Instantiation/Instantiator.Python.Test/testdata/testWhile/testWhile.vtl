@indent(indentation = 4, additional = 0)
@format(profile="python")
template testWhile(Configuration config, PythonCodeArtifact target) {
    def main(Configuration config, PythonCodeArtifact target) {
    	target.addWhile("420 == 69").addCall("print").addArgument('"420 == 69"');

		PythonCodeBlock b = target.addWhile("True");
    	b.addIf("420 >= 69").addBreak();
    	b.addIf("420 == 69").addContinue();
    }
}
