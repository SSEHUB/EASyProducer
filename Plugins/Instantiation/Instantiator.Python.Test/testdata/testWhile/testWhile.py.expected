
while 420 == 69:
    print("420 == 69")
while True:
    if 420 >= 69:
        break

    if 420 == 69:
        continue

